core-common/src/main/java/org/zalando/nakadi/repository/zookeeper/ZooKeeperHolder.java

```diff
@@ -115,4 +121,27 @@ private EnsembleProvider createEnsembleProvider() throws Exception {
                 throw new RuntimeException("Connection type " + conn.getType() + " is not supported");
         }
     }
+
+    private static class NakadiTracerDriver extends AdvancedTracerDriver {
+
+        private static final Logger LOG =
+                LoggerFactory.getLogger(NakadiTracerDriver.class);
+
+        @Override
+        public void addTrace(final OperationTrace trace) {
+            LOG.trace("curator trace: name:{} path:{} session:{} watch:{}",
```

Comment:
The intention is to set log level to TRACE for this particular class in the logger config, right?  Otherwise it's almost "no-op" and should have no measurable performance impact.

Reply:
yes